src/pybind/mgr/dashboard/frontend/src/app/shared/directives/permission-scope.directive.ts

```diff
@@ -0,0 +1,10 @@
+import { Directive, Input } from '@angular/core';
+
+@Directive({
+  selector: '[cdScope]'
+})
+export class PermissionScopeDirective {
```

Comment:
For the selector to be correct:
```suggestion
export class ScopeDirective {
```
This also requires you to change the folder and file names.